Library/Formula/libxcb.rb

```diff
@@ -0,0 +1,45 @@
+class Libxcb < Formula
+  desc "Modern base X11 library"
+  homepage "http://www.x.org"
+  url "http://www.x.org/releases/X11R7.7/src/xcb/libxcb-1.8.1.tar.gz"
+  sha256 "a171c660155dedf1fda9802a37abd04097820486525f4b6618a6203e9ce368a9"
+  head "git://anongit.freedesktop.org/git/xcb/libxcb"
+
+  depends_on "xproto"
+  depends_on "libxslt"
+  depends_on "pkg-config" => :build
+  depends_on :python
+
+  resource "xcb-proto" do
+    url "http://www.x.org/releases/X11R7.7/src/xcb/xcb-proto-1.7.1.tar.gz"
+    sha256 "a101cf4987609ca702cfefdbabcd48dde39511539459c284fbcda19e31394b9f"
+  end
+
+  resource "pthread-stubs" do
+    url "http://www.x.org/releases/X11R7.7/src/xcb/libpthread-stubs-0.3.tar.gz"
+    sha256 "3031f466cf0b06de6b3ccbf2019d15c4fcf75229b7d226a711bc1885b3a82cde"
+  end
+
+  def install
+    resource("pthread-stubs").stage do
+      system "./configure", "--prefix=#{buildpath}/pthread-stubs"
+      system "make", "install"
+    end
+
+    resource("xcb-proto").stage do
+      system "./configure", "--prefix=#{buildpath}/xcb-proto"
+      system "make", "install"
+    end
```

Comment:
It looks like I can probably combine these two into a loop; I'll try that later.